packages/unit-testing/README.md

```diff
@@ -36,35 +36,35 @@ However, if you want to configure SuiteCloud Unit Testing manually, do the follo
 3. To initialize the NPM package, from the root of your SDF project folder, run `npm init`.
 >💡 A `package.json` file is created in your SDF project folder.
 4. In your `package.json` file, add the following code:
-```json
-{
-  "scripts": {
-    "test": "jest"
-  }
-}
-```
+    ```json
+    {
+    "scripts": {
```

Comment:
Should we add one more tab here?

Reply:
I added only one because it's a small file, but we can also have two.